cosign.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: cosign
+  version: 1.13.1
+  description: Container Signing
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+
+environment:
+  contents:
+    keyring:
+      - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub

```

Comment:
`keyring`/`repositories` are redundant here.

Reply:
i dont need to use keyring if repository is there? i saw the other packages they all have both.

Reply:
Some do but it is a problem for a few reasons so trying to avoid it